modules/installation-special-config-kmod.adoc

```diff
@@ -0,0 +1,304 @@
+deployed in a RHEL container. Keep in mind that modules are rebuilt
+automatically on each node when that node gets a new kernel. For that
+reason, each node needs access to a yum repository that contains the
+kernel and related packages needed to rebuild the module. That content
+is best provided with a valid RHEL subscription.
+====
+
+== Build and test the kernel module container
+Gather the kernel module’s source code, the KVC framework, and the
+kmod-via-containers software. Then build and test the module. To do
+that on a RHEL 8 system, do the following:
+
+Procedure
+
+. Get a RHEL 8 system, then register and subscribe it:
++
+----
+# subscription-manager register
+Username: yourname
+Password: ***************
+# subscription-manager subscribe --auto
+----
+
+. Install software needed to build the software and container:
++
+----
+# yum install podman make git -y
+----
+
+. Clone the kmod-via-containers repository:
++
+----
+$ mkdir kmods; cd kmods
+$ git clone https://github.com/dustymabe/kmods-via-containers.git
+----
+
+. Install a KVC framework instance on your build host to test the module.
+This adds a kmods-via-container systemd service and loads it:
++
+----
+$ cd kmods-via-containers/
+$ make install
+$ sudo systemctl daemon-reload
+----
+
+. Get the kernel module. This could be a third-party module that you don’t
+have control over, but is supplied by others. You will need configuration files
+and source code similar to those shown in the kvc-simple-kmod example that can
+be cloned to your system as follows:
++
+----
+$ cd ..
+$ git clone https://github.com/dustymabe/kvc-simple-kmod
+----
+
+. Edit the configuration file (`simple-kmod.conf`, in his example) and
+change the name of the Dockerfile to Dockerfile.rhel so the file appears as
+shown here:
++
+----
+$ cd kvc-simple-kmod
+$ cat simple-kmod.conf
+
+KMOD_CONTAINER_BUILD_CONTEXT="git://github.com/dustymabe/kvc-simple-kmod.git"
+KMOD_CONTAINER_BUILD_FILE=Dockerfile.rhel
+KMOD_SOFTWARE_VERSION=dd1a7d4
+KMOD_NAMES="simple-kmod simple-procfs-kmod"
+----
+
+. Create an instance of kmods-via-containers@.service for your kernel module
+(simple-kmod in this example) and enable it:
++
+----
+$ sudo make install
+$ sudo kmods-via-containers build simple-kmod $(uname -r)
+----
+. Enable and start the systemd service, then check the status:
++
+----
+$ sudo systemctl enable kmods-via-containers@simple-kmod.service
+$ sudo systemctl start kmods-via-containers@simple-kmod.service
+$ sudo systemctl status kmods-via-containers@simple-kmod.service
+● kmods-via-containers@simple-kmod.service - Kmods Via Containers - simple-kmod
+   Loaded: loaded (/etc/systemd/system/kmods-via-containers@.service;
+          enabled; vendor preset: disabled)
+   Active: active (exited) since Sun 2020-01-12 23:49:49 EST; 5s ago...
+----
+
+. Here are a few ways to interact with the kernel modules in this example and make sure they were loaded:
```

Comment:
s/make sure/ensure

Reply:
I'll use that sentence to introduce it. Otherwise, I'll leave the commands as they are.